pages/[...path].js

```diff
@@ -61,6 +61,7 @@ export async function getServerSideProps(ctx) {
       },
     }
   } else {
+    ctx.res.statusCode = res.res.status
```

Comment:
What is this for exactly.. I feel like this will cause some unwanted side effects..